Various improvements

Digitalone1 opened the pull request:
- Improve `populatePluginsStack` readability returning early if plugins list is empty.
- Added a new option to show fixed buttons in plugins list. A new RowLayout is added and the one to be shown is decided by the database setting. I had also to make new functions for `onTriggered`/`onClicked` signals. @wwmm It works to me, but please test it and ensure the callback functions are working good. Related to #4325.
- RNNoise models list is not shown properly when entry names are too long. I had to add the fillWidth Layout to restrict it inside the container. Also added `elide` and `wrapMode` properties to the entries of other lists so we are sure the text is not overflowing if too long.
- Changed back the CTRL+T shortcut to CTRL+Q. Thanks to ChatGPT I find the `GlobalDrawer` QML class to catch the CTRL+Q quit event from Gnome, so now it terminates the service rather than closing only the window. @wwmm Please test it on your system. You might also consider to remove the previous CTRL+Q `Shortcut` class since there's the duplicate in `GlobalDrawer` (maybe the two conflicts and quit the application two times on KDE). Fixes #4400.

gwuen commented:
The new option is shown in the Background Service section, I guess this was not the intention?

Digitalone1 commented:
I tried to remove the Shortcut class with `CTRL+Q`, deleted the `StandardKey.Quit` and added `CTRL+Q` in the GlobalDrawer:

```
        actions: [
            Kirigami.Action {
                shortcut: "Ctrl+Q"
                onTriggered: {
                    Qt.quit();
                }
            }
        ]
```

And it works either! It seems the Shortcut class has some incompatibility with Gnome.

What do you want to do? I suggest to move all the shortcuts in the GlobalDrawer. It seems more appropriate since the GlobalDrawer is a Kirigami class while Shortcuts if from Qt Quick.
